curriculum/challenges/english/20-upcoming-python/learn-algorithm-design-by-building-the-shortest-path-algorithm/655791e6cf5e03be3de73451.md

```diff
@@ -0,0 +1,65 @@
+---
+id: 655791e6cf5e03be3de73451
+title: Step 34
+challengeType: 20
+dashedName: step-34
+---
+
+# --description--
+
+If you try to uncomment your function call, it won't work. You have a couple of bug to fix. The first one happens because in the nested `if` you are trying to access `paths[node][-1]` but first you need to be sure that `paths[node]` is not empty.
```

Comment:
```suggestion
If you try to uncomment your function call, it won't work. You have a couple of bugs to fix. The first one happens because in the nested `if` you are trying to access `paths[node][-1]` but first you need to be sure that `paths[node]` is not empty.
```